pkg/tests/handlers/accounts.go

```diff
@@ -226,12 +227,23 @@ func TestClusterAuthorizations(ctx context.Context, options *types.TestOptions)
 	return nil
 }
 
+func randomizeResourceName(ResourceName_list [5]string) string {
+	ResourceName := fmt.Sprint(ResourceName_list[rand.Intn(len(ResourceName_list))])
+	return ResourceName
+}
+
+func randomizeCount(min_count int, max_count int) int {
+	Count := min_count + rand.Intn(max_count-min_count+1)
+	return Count
+}
+
 func clusterAuthorizationsBody(ctx context.Context, clusterID string, options *types.TestOptions) []byte {
+	rand.Seed(time.Now().UnixNano())
 	buff := &bytes.Buffer{}
 	reservedResource := v1.NewReservedResource().
-		ResourceName(helpers.M5XLargeResource).
+		ResourceName(randomizeResourceName(helpers.AWSResources)).
```

Comment:
I don't see the need to pass a parameter to the function, the resources are predefined in a slice that we are not changing. You can access that list directly inside the function without passing any parameter, since it is a publicly accessible value and you are not doing any modifications to it.